{"uid":"39662e87e75a4e17","name":"An authenticated Service Provider cannot send an RTP to an unactivated debtor","fullName":"A Creditor Service Provider sends an RTP through API: An authenticated Service Provider cannot send an RTP to an unactivated debtor","historyId":"e840fb9f4b291411de96677fad528737","time":{"start":1737443242542,"stop":1737443244706,"duration":2164},"status":"failed","statusMessage":"AssertionError\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"bdd-tests/steps/send_rtp_steps.py\", line 32, in then_the_rtp_is_not_created\n    assert context.latest_rtp_response.status_code == 422\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"AssertionError\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"bdd-tests/steps/send_rtp_steps.py\", line 32, in then_the_rtp_is_not_created\n    assert context.latest_rtp_response.status_code == 422\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[{"name":"Given the debtor A has fiscal code random","time":{"start":1737443242542,"stop":1737443242543,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"And the debtor Service Provider is authenticated","time":{"start":1737443242543,"stop":1737443243472,"duration":929},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Given the creditor Service Provider is authenticated","time":{"start":1737443243472,"stop":1737443244285,"duration":813},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"When the creditor Service Provider sends an RTP to the debtor A","time":{"start":1737443244285,"stop":1737443244704,"duration":419},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Then the RTP is not created because the debtor is not activated","time":{"start":1737443244704,"stop":1737443244705,"duration":1},"status":"failed","statusMessage":"AssertionError\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.12.8/x64/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"bdd-tests/steps/send_rtp_steps.py\", line 32, in then_the_rtp_is_not_created\n    assert context.latest_rtp_response.status_code == 422\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":5,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"tag","value":"authentication"},{"name":"tag","value":"send"},{"name":"tag","value":"unhappy_path"},{"name":"feature","value":"A Creditor Service Provider sends an RTP through API"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":2,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":2},"items":[{"uid":"1d4d091882c80fa2","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/166//#testresult/1d4d091882c80fa2","status":"failed","statusDetails":"AssertionError\n","time":{"start":1737389836713,"stop":1737389838842,"duration":2129}}]},"tags":["send","unhappy_path","authentication"]},"source":"39662e87e75a4e17.json","parameterValues":[]}